components/ui/Input.tsx

```diff
@@ -41,13 +44,21 @@ const Input = ({
           selectionColor={Colors.darkGray}
           underlineColorAndroid="transparent"
         />
-        {value && (
-          <AntDesign
-            testID="close"
-            name="close"
+        {testID == 'password' ? (
```

Comment:
```suggestion
        {type == 'password' ? (
```

Create a prop type that will have types of input:
text, password, number
keep text by default